pkg/controller/stacks/stack/stack.go

```diff
@@ -589,6 +590,16 @@ func (h *stackHandler) prepareDeployment(d *apps.Deployment) {
 	d.Spec.Template.SetName(name)
 	meta.AddLabels(&d.Spec.Template, matchLabels)
 	d.Spec.Selector = &metav1.LabelSelector{MatchLabels: matchLabels}
+	d.Spec.Template.Spec.ImagePullSecrets = h.ext.Spec.Controller.ImagePullSecrets
```

Comment:
one day i want to take a pass and remove all these lingering `ext` references in the code/structs. it's a lingering reminder of when stacks were called extensions :(